chapters/ch04.md

```diff
@@ -35,18 +53,26 @@ It was as if the older algorithms didn’t know what to do with all the data we
 
 ->
 
+Như thể thuật toán cổ điển không biết xử lý ra sao với tất cả lượng dữ liệu đang có. 
+
+
+
 
 If you train a small neutral network (NN) on the same supervised learning task, you might get slightly better performance:
 
 ->
 
+Nếu bạn huấn luyện một mạng neural nhỏ cho cùng một tác vụ học có kiểm soát, có thể bạn sẽ đạt chất lượng cao hơn một chút
+
 
 ![img](../imgs/C04_02.png)
 
 Here, by “Small NN” we mean a neural network with only a small number of hidden units/layers/parameters. Finally, if you train larger and larger neural networks, you can obtain even better performance [1]:
 
 ->
 
+"Mạng neural nhỏ" ở đây có nghĩa là mạng neural với ít các nút ẩn/tầng/tham số. Và sau cùng, càng huấn luyện mạng neural to càng cho kết quả tốt hơn [1]: 
+
```

Comment:
* thêm từ "nếu bạn" cho "càng huấn luyện mạng"
* " càng huấn luyện mạng neural to càng cho kết quả tốt hơn" -> "lớn hơn" -> maybe là "nếu bạn càng huấn luyện mạng neural lớn hơn bao nhiêu thì bạn càng có thể đạt được kết quả tốt hơn bấy nhiêu"

Reply:
> * " càng huấn luyện mạng neural to càng cho kết quả tốt hơn" -> "lớn hơn" -> maybe là "nếu bạn càng huấn luyện mạng neural lớn hơn bao nhiêu thì bạn càng có thể đạt được kết quả tốt hơn bấy nhiêu"

Mình thấy dịch thế này chưa sát. Như câu của bạn dễ bị hiểu là mạng lớn tới vô hạn thì kết quả cũng tốt lên vô hạn.

Mình đề xuất:
"Cuối cùng, nếu bạn huấn luyện mạng lớn hơn nữa, bạn thậm chí có thể đạt được kết quả tốt hơn."

Reply:
thế này thì sao nhỉ? 

> "Mạng neural nhỏ" ở đây có nghĩa là mạng neural với ít nút ẩn/tầng/tham số. Sau cùng, bạn có thể cải thiện chất lượng thêm nữa nếu dùng các mạng neural lớn hơn.